### DISCUSSION TITLE
Importing postman collections doesn't import Tests/Pre Request scripts

### DISCUSSION THREAD

**Discussion by ifnotnithya** [thunderclient/thunder-client-support#1289] [Q&A]
Hi

I am testing postman collection import into thunder client. 
I see that the, code I have in Pre Request scripts and Tests are not imported into Thunder Client.
Is this by design?

Thanks

**Comment by rangav** [comment]
Hi @ifnotnithya Yes by design 

the  js code file is separate from collection files.

**Reply by ifnotnithya** [reply]
Ah ok. 
Just to understand this better.
- Is there a separate utility to do this import? If so, can you point me to the documentation for this.
- My current code in PreReq / Tests is not a combination of both js functions and/or getting/setting values into env variables
Or 
- we need to rewrite the same functionality that we have in postman prereq/tests in TC format. I have figured this out, but this is lots of rewrite.

**Reply by rangav** [reply]
Currently we don't support postman import of scripts.

but you may find scrpting very similar.

lot of the functionality is done by UI in TC.

scripting docs
https://github.com/rangav/thunder-client-support/blob/master/docs/filters.md#custom-filters

Assertions using scripting 
https://github.com/rangav/thunder-client-support/releases/tag/v2.10.0

**Reply by ifnotnithya** [reply]
Got it. Thx.
